.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ jobs:
   build-core-27:
       <<: *build-core
       docker:
-        - image: circleci/python:2.7-stretch-node-browsers
+        - image: circleci/python:2.7.16-stretch-node-browsers
```

Comment:
From the previous CI runs, this seems unwarranted. The tests seemed to be passing fine and not requiring a specific version override atm.

Reply:
Ah, OK, will revert.

Reply:
fixed in c39de33